gremlin-dotnet/example/example.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Gremlin.Net" Version="3.7.0" />

```

Comment:
Changed version to 3.6.5

Reply:
There may be some trickery which can be done by configuring multiple instances of dependabot to run in each branch along with some rules to restrict the version in each branch. That would still require the committer to be careful when merging those commits up to other branches to preserve the correct version. If we consider it essential to have the example versions align with the latest release in each respective branch, this might be best left as an additional administrative task for release managers. It may be possible to use the Maven Replacer Plugin to automatically this version with every `mvn compile` but the same caution around merging between branches would apply.

Personally I'm somewhat indifferent to how/if the versions should be maintained here. The main value I see in these examples is demonstrating basic driver setup, connection, and querying. This has remained relatively constant over the last few major releases and I think that a 3.7.x example is still useful for someone attempting to use 3.6.x.

Reply:
I'm thinking it might be fine to let Dependabot handle it since the examples are quite simple and would probably still work with future updates.  We could maybe implement tests that execute during the maven build at some point to catch breaking changes, which would avoid adding extra overhead for release managers.

Reply:
> Personally I'm somewhat indifferent to how/if the versions should be maintained here. The main value I see in these examples is demonstrating basic driver setup, connection, and querying. This has remained relatively constant over the last few major releases and I think that a 3.7.x example is still useful for someone attempting to use 3.6.x.

You're probably right and in addition to that, I'm not sure if it's worth any effort to maintain different examples for the different versions. Most users will just use the latest version any way. That's at least the case for .NET according to the NuGet download statistics.
So, from my side, we can stay on 3.7.0 here.